com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/RaytracingRenderer.raytrace

```diff
@@ -2,7 +2,11 @@
 #pragma max_recursion_depth 10
 
 // WARNING: This define must be kept in sync with the c# code
+#if defined(SHADER_API_PSSL)
+#define RAYTRACING_MAX_RECURSION 1
```

Comment:
why this one only? RayTracingReflection.raytracing and RaytracingIndirectDiffuse need to have their max recursion force to 1 aswell

Reply:
@anisunity I am fine with you reworking the wizard code. Good point about RaytracingRenderer.raytrace - mainly it was done there as a reaction to some HDRP_DXR tests e.g. 902_Materials_SG_Variants_xxxxx where a third of the screen is testing recursive raytrace, in the case of PS5 I wanted to achieve a consistent result of just the background. I will rethink best way to do this.

Reply:
@anisunity I have reverted this change to RayTracingRenderer.raytrace, and added a scripting define in ProjectSettings NO_RAY_RECURSION, and tests to set the max recursion in the shader setup .cs code for this and the other cases